assets/app/views/settings.html

```diff
@@ -65,22 +65,53 @@
                 </div>
 
                 <div ng-repeat="quota in quotas" class="gutter-bottom">
-                  <div ng-if="quote.status.hard.cpu || quota.status.hard.memory">
-                    <h2>
-                      Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
-                    </h2>
-                    <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <h2>
+                    Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
+                  </h2>
+                  <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <dl ng-if="quota.spec.scopes.length" class="left">
+                    <dt>Scopes:</dt>
+                    <dd>
+                      <div ng-repeat="scope in quota.spec.scopes">
+                        {{scope | humanizeQuotaScope}}
+                        <span class="text-muted small">
+                          &mdash;
+                          <span ng-switch="scope" class="hide-ng-leave">
+                            <span ng-switch-when="Terminating">
+                              Matches all pods that have an active deadline.
```

Comment:
Yeah i'm ok with that being different for 3.2

@derekwaynecarr think we can get the CLI wording updated in 1.3?  "Applies to pods that have..."